provider/aws/app.go

```diff
@@ -24,6 +24,9 @@ func (p *Provider) AppCreate(name string, opts structs.AppCreateOptions) (*struc
 
 	res, err := p.ECR.CreateRepository(&ecr.CreateRepositoryInput{
 		RepositoryName: aws.String(fmt.Sprintf("%s%s", p.RepositoryPrefix(), name)),
+		ImageScanningConfiguration: &ecr.ImageScanningConfiguration{
+			ScanOnPush: aws.Bool( p.ecr_scan_on_push_enable == "true"),
```

Comment:
use the ecr scan variable as bool instead of string, and do the check on the var assignment